.github/workflows/publish-prisma-fmt-wasm.yml

```diff
@@ -0,0 +1,78 @@
+name: Build and publish @prisma/prisma-fmt-wasm
+
+on: workflow_dispatch # triggered manually for now
+
+jobs:
+  build:
+    name: Build and publish @prisma/prisma-fmt-wasm
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2.3.4
+
+    # Install nix
+    # Copy-pasted from https://github.com/marketplace/actions/install-nix
+    - uses: cachix/install-nix-action@v14
+      with:
+        install_url: https://nixos-nix-install-tests.cachix.org/serve/vij683ly7sl95nnhb67bdjjfabclr85m/install
+        install_options: '--tarball-url-prefix https://nixos-nix-install-tests.cachix.org/serve'
+        extra_nix_config: |
+          experimental-features = nix-command flakes
+          access-tokens = github.com=${{ secrets.GITHUB_TOKEN }}
+
+    - name: Print input
+      env: { THE_INPUT: '${{ toJson(github.event.client_payload) }}' }
+      run: |
+        echo $THE_INPUT
+
+    # Update the cargo hash
+    - run: |
+        cd ./build-prisma-fmt
+        # Update dependencies (including engines).
+        nix run .#cargo update
+        # Set up build to fail and output the new hash.
+        sed -i 's/cargoSha256.*/cargoSha256 = fakeSha256;/' flake.nix
+        # Extract the new dependencies source hash
+        CARGO_HASH=`nix build 2>&1 > /dev/null | grep 'got:.*sha256' | sed 's/.*got:.*sha256/sha256/'`
+        echo "New cargo hash: $CARGO_HASH"
+        # Put the new cargo hash in flake.nix
+        sed -i "s/cargoSha256.*$/cargoSha256 = \"$CARGO_HASH\";/" flake.nix
+
+    # Build
+    - run: nix build ./build-prisma-fmt
+
+    # Update the npm package version
+    - run: |
+        cd ./build-prisma-fmt
+        nix run .#npm version patch
```

Comment:
I assume we'll have to use something from the dispatch input if we want to sync the versions with the `dev-...` and stable releases. I think I can't make that work without merging first, however.

Reply:
What's the version returned by this?

Reply:
We're starting at `0.1.0`, so in the current state, it would go `0.1.1`, `0.1.2`, etc.

Reply:
I assume we want to be in sync with the other npm versions, which should be possible, but not before I can iterate on the GitHub action and see what happens.

Reply:
Makes sense

Reply:
So this is a temporary version, that we would later replace with using the same version as for all other packages in this repo?